node/test/Test/Pos/Explorer/Identity/BinarySpec.hs

```diff
@@ -6,13 +6,14 @@ module Test.Pos.Explorer.Identity.BinarySpec
 
 import           Universum
 
-import           Test.Hspec     (Spec, describe)
+import           Test.Hspec            (Spec, describe)
+import           Test.Hspec.QuickCheck (modifyMaxSuccess)
 
-import           Pos.Explorer   (TxExtra)
-import           Test.Pos.Util  (binaryTest)
+import           Pos.Explorer          (TxExtra)
+import           Test.Pos.Util         (binaryTest)
 
 
 spec :: Spec
 spec = describe "Explorer types" $ do
     describe "Bi instances" $ do
-        binaryTest @TxExtra
+        modifyMaxSuccess (const 100) $ binaryTest @TxExtra
```

Comment:
I guess 100 is the default and I don't see a good reason why we need to modify max success for this particular type (is it structurally big?), so it looks a bit confusing to me.